training-maquetado/scss/detail.scss

```diff
@@ -10,6 +10,16 @@
 
 .book-cover {
   margin-right: 52px;
+
+  &::before {
+    position: absolute;
```

Comment:
you need to set the position of the book cover to relative so this element is position relative to the book cover and not the page